### PR TITLE
Add previous files

### PR DESCRIPTION
**Pull request by OscarFMdev**
## PR for peer-to-peer code review

In this branch I:
- Copy and paste the code of the previous project to get a peer-to-peer code review.

### Thank you for helping me improve! 🙌

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  for (let i = 0; i < allCheckBoxes.length; i += 1) {
+    allCheckBoxes[i].addEventListener('click', (e) => {
+      checkBoxesStatus(e);
+    });
+  }
+});
+
+const deleteAll = document.querySelector('.reload-icon');
+deleteAll.addEventListener('click', () => {
```
Comment (AlejandraKeber) [inline src/index.js]: It is a good coding practice to put all declarations at the top of each script or function. This give cleaner code and provide a single place to look for local variables

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <div id="container">
+      <div class="heading">
+        <h2 class="list-title">Today's To Do</h2>
+        <i class="fa-solid fa-rotate reload-icon icon"></i>
+      </div>
+      <hr class="line-separator">
+
+      <!-- Input -->
+      <form action="">
+        <input class="new-task" type="text" placeholder="Add to your list..." required>
```
Comment (AlejandraKeber) [inline src/index.html]: You have a form without any action in your html code.   This link is a helpful tool to check your code: https://validator.w3.org/nu/#textarea

**Review by AlejandraKeber — CHANGES_REQUESTED** [review]
Hi @OscarFMdev  🙋‍♂️,
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

**Highlights: 💪 **
- Keep all the JavaScript code in JS files, not in the script tag ✅
- Keep your code clean by following type checks and naming ✅
- You add node_modules dir to your .gitignore file  ✅
- You didn't use console.log nor windows.alert ✅


**Proposed changes: ♻️ **
Based on js best practices I recommend you some changes to have a clean code (please see the comments below)

File: src/index.js (hunk 1 above)
Comment (OscarFMdev) [reply src/index.js]: @AlejandraKeber thank you, I didn't notice!

File: src/index.html (hunk 2 above)
Comment (OscarFMdev) [reply src/index.html]: Nice link, didn't know that existed, thank you! @AlejandraKeber  🙌

**Review by AlejandraKeber — APPROVED** [review]
Hi @OscarFMdev, **Great job!!!** 🥇
## STATUS: Approved 🎉
You've completed the requirements, there's nothing else to say than merge it! 

Congratulations! 🎉

**Pull request merged**